src/utils.jl

```diff
@@ -244,33 +244,6 @@ Returns true if `typemax(T)` of a type `T <: Integer` exists.
 isbounded(::Type{T}) where {T <: Integer} = isconcretetype(T)
 isbounded(::Type{BigInt}) = false
 
-"""
-    deepcopy_adjlist(adjlist::Vector{Vector{T}})
-
-Internal utility function for copying adjacency lists.
```

Comment:
This is specific to SimpleGraphs so was moved to the SimpleGraphs module, in utils.jl.